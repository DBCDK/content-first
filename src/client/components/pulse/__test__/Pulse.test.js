import React from 'react';
import renderer from 'react-test-renderer';
// import {mount} from 'enzyme';

import Pulse from '../Pulse.component';

const pid = '870970-basis:52038014';
const position = {x: 26, y: 46};
const description = 'lorem ipsum . . .';

describe('Pulse', () => {
  it('renders initial component', () => {
    const tree = renderer
      .create(
        <Pulse
          onClick={() => {
            this.rollOverTrigger(pid, description, position, 0);
          }}
          delay={2}
          position={position}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  // it('calls onclick prop on click', () => {
  //   const tree = mount(
  //     <Pulse
  //       onClick={() => {
  //         this.rollOverTrigger(pid, description, position, 0);
  //       }}
  //       position={position}
  //     />
  //   );
  //   tree.find('.pulse-toucharea').simulate('click');
  // });
});
