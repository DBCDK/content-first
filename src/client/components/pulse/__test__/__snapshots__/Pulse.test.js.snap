// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pulse renders initial component 1`] = `
<div
  className="pulse-toucharea  pulse-active"
  onClick={[Function]}
  style={
    Object {
      "left": "26%",
      "top": "46%",
    }
  }
>
  <div
    className="pulse"
  />
</div>
`;
